pkg/ui/workspaces/cluster-ui/src/icon/refreshIcon.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import * as React from "react";
+
+interface IconProps {
+  className?: string;
+  onClick?: () => void;
```

Comment:
Looks like you can remove this since you put onClick into the container div instead (which IMO is preferable over putting it on the svg component).